packages/page-democracy/src/Overview/Referendum.tsx

```diff
@@ -179,4 +193,27 @@ export default React.memo(styled(Referendum)`
       text-align: center;
     }
   }
+
+  * {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+} 
+
+.progress-container {
+  position: relative;
+  width: 250;
+ 
+}
+
+.progress-centered-element {
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 250;
+
+}
```

Comment:
I'm convinced this is due to the progress component, at least in this case really don't think we should be using `*` rules unless absolutely needed.